integrations/tensorflow/iree-dialects/lib/Dialect/PyDM/IR/PyDMDialect.cpp

```diff
@@ -64,7 +64,7 @@ Operation *IREEPyDMDialect::materializeConstant(OpBuilder &builder,
     return builder.create<PYDM::SuccessOp>(loc, type);
   }
 
-  llvm_unreachable("unhandled iree_pydm constant materialization");
+  assert(false && "unhandled iree_pydm constant materialization");
```

Comment:
wouldn't that cause a warning since this path doesn't return an Operation*? One convenient part of `llvm_unreachable` is that it allows not returning without getting compiler warnings.

Reply:
indeed i need to add a `return nullptr;` here.

Re "convenient" please read my explanation on this topic in the PR description and let me know what you think?

Reply:
We are looking at the diff from only one commit here. This is fixed by the subsequent "manual fixes" commit, by adding a `return nullptr`. Re "convenient", WDYT about the explanation on this topic in this PR's description?

Reply:
Wait, GitHub is having a bug here. Even though I selected "Changes from all commits", it's only showing the first commit's changes. WTF? @GMNGeoffrey

Reply:
It's because Thomas's comment is on only the first commit, I think. Yes GitHub's PR review interface is garbage

Reply:
ah sorry I hadn't read the description... I don't have a strong opinion either way but the rational sounds reasonable to me.

Reply:
Wait, so as soon as any reviewer comments on one specific commit's diff, it changes the behavior of the "Changes from all commits" diff view for everyone else to silently show only this commit's diff ?!

Reply:
Ah, at least it only affects that one file that was commented on.

Reply:
Oh wait, no this is different. There's something weird when you follow the link from a comment, but in fact this is showing the latest version AFAICT. There isn't a return here at this commit: https://github.com/google/iree/blob/7a9de501cc6d984030cdb6febff7e1b9bf84d861/integrations/tensorflow/iree-dialects/lib/Dialect/PyDM/IR/PyDMDialect.cpp#L67

Reply:
What?! There is the return, at this commit, in my local git clone.
And GitHub sees it being added in the previous commit [here](https://github.com/google/iree/pull/8521/commits/94bbefcabe122de8125343e61991f18181137651).

Reply:
Also, the fact that CI passed confirms that the return is there in the top commit --- CI was definitely failing before I pushed these fixes, and I found that occurence (along with others) from the CI failure log.

Reply:
These are different files. The copy under the IREE base, and the copy under integrations/tensorflow

Reply:
This is not the first time that having two copies of these files has caused confusion...

Reply:
Ah got it! Sorry @ThomasRaoux and @GMNGeoffrey for not properly reading the full path.